andela/travel_tool_deployment_scripts
# #159817143 Setup Backend K8s Objects

#### What does this PR do?
- Adds blocking job config object for the backend
- Adds deployment config object for the backend
- Adds secret config object for the backend
- Adds service config object for the backend
- Adds production ingress object for the backend
- Share horizontal pod autoscaler object for the backend

#### Description of Task to be completed?
Setup K8s deployment Objects for Travel frontend
Travel tool will be deployed to K8s managed cluster. It will require K8s Objects that will define all the resources that the application will require.

We will be using reactiveops K8s deployment scripts for this setup.

Acceptance critetia

Should contain the following objects
- Deployment
- Service
- Ingress
- Blockingjob
- Secret

#### How should this be manually tested?
1. Create a cluster on GCP and make it accessible locally
2. Build travel tool backend image and push it to GCR
3. Set variables to correct values in **_bin/test_**  file
4. Run this command: `bin/test build`
5. Deploy the application using this command: `k8s-deploy-and-verify -f deploy/travella-backend.config`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#159817143
